hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/HapiTestEngine.java

```diff
@@ -504,6 +507,9 @@ public HapiTestEngineExecutionContext execute(
                 }
             }
 
```

Comment:
to ensure that most of the state changes will be flushed we probably could sleep here for some time